packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,53 @@
+import { recipe } from "../vanilla-extract";
+
+export const textAreaBoxRecipe = recipe({
```

Comment:
let's rename this to 
```suggestion
export const textarea = recipe({
```